core/src/main/resources/windows-service/jenkins-slave.xml

```diff
@@ -23,15 +23,22 @@ THE SOFTWARE.
 -->
 
 <!--
-  Windows service definition for Jenkins slave
+  Windows service definition for Jenkins agent.
+  The actual version of this definition has been moved to https://github.com/jenkinsci/windows-slave-installer-module/blob/master/src/main/resources/org/jenkinsci/modules/windows_slave_installer/jenkins-slave.xml.
```

Comment:
Left behind from https://github.com/jenkinsci/jenkins/blob/f79f53ffbdff78c8bb4fefb51c6acbb3f266001b/core/src/main/java/hudson/lifecycle/WindowsSlaveInstaller.java#L219 it seems -- probably time to remove it from core?